mesheryctl/internal/cli/root/system/model.go

```diff
@@ -446,6 +446,12 @@ mesheryctl exp model view [model-name]
 		}
 		return nil
 	},
+	Args: func(cmd *cobra.Command, args []string) error {
+		if len(args) == 0 {
+			return cmd.Help()
```

Comment:
@lekaf974 Ensure error handling consistency: Consider returning errors for failed cmd.Help() calls and provide descriptive messages for better debugging.